app/experimenter/static/rapid/components/forms/ExperimentFormOptions.tsx

```diff
@@ -0,0 +1,9 @@
+export const audienceOptions = [
+  { label: "AUDIENCE 1", value: "AUDIENCE 1" },
+  { label: "AUDIENCE 2", value: "AUDIENCE 2" },
+];
+
+export const featureOptions = [
+  { label: "FEATURE 1", value: "FEATURE 1" },
+  { label: "FEATURE 2", value: "FEATURE 2" },
+];
```

Comment:
Is the plan that these would be hardcoded in Experimenter, or eventually be pulled from the nimbus-shared module?

Oh, nevermind. Now I remember seeing some recent PRs in https://github.com/mozilla/nimbus-shared/pull/54 and https://github.com/mozilla/nimbus-shared/pull/55 for adding Audiences and Features.

Reply:
yup yup, just temporary